docs/predicate/slsa/v2.md

```diff
@@ -0,0 +1,621 @@
+<!--
+---
+linkTitle: "Chains:v2"
+weight: 10
+---
+-->
+
```

Comment:
Suggest adding a link to the official slsa 1.0 spec https://slsa.dev/spec/v1.0/provenance somewhere at the top of this file.